toil/responses.py

```diff
@@ -0,0 +1,57 @@
+from enum import StrEnum
+from typing import Any, Optional
+
+from pydantic import BaseModel
```

Comment:
In VSCode: `No name 'BaseModel' in module 'pydantic'pylint(no-name-in-module)`

Reply:
Strangely I'm not getting that but considering other errors like 
```
workflows/views.py:50:19: E1101: Class 'Workflow' has no 'objects' member (no-member)
```
I think pylint may be hopeless busted for this project anyway.

Reply:
There's also `pyright` which I think have in the makefile as well that's something between a type checker and a linter which might be a good compromise to replace mypy and pylint but I'm not sure how much ground each covers.